circle.yml

```diff
@@ -2,7 +2,7 @@ general:
   branches:
     ignore:
       # We only want to build pull requests for testing. If something is merged,
-      # then we are prepping for release an there is no need to build it again.
+      # then we are prepping for release and there is no need to build it again.
```

Comment:
Went ahead and cherry-picked this commit into PR ( https://github.com/conda-forge/staged-recipes/pull/2144 ). Please rebase to clear this out of this PR.